diff --git a/setup_env.py b/setup_env.py
index 8011872..8b97ea9 100644
--- a/setup_env.py
+++ b/setup_env.py
@@ -83,16 +83,14 @@ def run_command(command, shell=False, log_step=None):
         log_file = os.path.join(args.log_dir, log_step + ".log")
         with open(log_file, "w") as f:
             try:
-                subprocess.run(command, shell=shell, check=True, stdout=f, stderr=f)
+                subprocess.run(command, shell=shell, stdout=f, stderr=f)
             except subprocess.CalledProcessError as e:
                 logging.error(f"Error occurred while running command: {e}, check details in {log_file}")
-                sys.exit(1)
     else:
         try:
-            subprocess.run(command, shell=shell, check=True)
+            subprocess.run(command, shell=shell)
         except subprocess.CalledProcessError as e:
             logging.error(f"Error occurred while running command: {e}")
-        sys.exit(1)
 
 def prepare_model():
     _, arch = system_info()
@@ -229,4 +227,4 @@ if __name__ == "__main__":
     args = parse_args()
     Path(args.log_dir).mkdir(parents=True, exist_ok=True)
     logging.basicConfig(level=logging.INFO)
-    main()
\ No newline at end of file
+    main()
